content/md/what-is/understand-data-quality.md

```diff
@@ -0,0 +1,88 @@
+---
+id: understand-data-quality
+themes: first-steps
+title: Understand **data quality**
+popular: false
+ee-only: true
+related: improve-data-quality
+---
+
+# About data quality
```

Comment:
I don't understand why you have the same content below, the 1st article should be also visible in What is, no need to duplicate the content.

Reply:
I changed it this morning so it should be resolved now